protocols/isis/server/lsp.go

```diff
@@ -39,8 +39,8 @@ func (s *Server) generateLocalLSP() *packet.LSPDU {
 			packet.NewAreaAddressesTLV(s.areaIDs()),
 			s.getProtocolsSupportedTLV(),
 			packet.NewIPInterfaceAddressesTLV(s.netIfaManager.getAddressesIPv4()),
-			// TODO: Add ExtendedISReachabilityTLV
-			// TODO: Add ExtendedIPReachabilityTLV
+			s.ExtendedIPReachabilityTLV(),
```

Comment:
Is there a reason why one is exported and the other isn't?

Reply:
No. Just me being inattentive. Fixed.